### PR TITLE
OCM-8053 | test: Automated id:43252 validating inputs for creating spot machinepools

### PR DESCRIPTION
**Pull request by jerichokeyne**
output:
```
$ CLUSTER_ID=2b99vnpc3u6ktad4ad28ana0m67o7j7f ginkgo -focus 43252 tests/e2e/
Running Suite: e2e tests suite - /home/jericho/work/repos/rosa/tests/e2e
========================================================================
Random Seed: 1715798790

Will run 1 of 67 specs
SSSSSSSSS•SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS

Ran 1 of 67 Specs in 4.411 seconds
SUCCESS! -- 1 Passed | 0 Failed | 0 Pending | 66 Skipped
PASS

Ginkgo ran 1 suite in 5.371714118s
Test Suite Passed

```

### CODE REVIEW

File: tests/e2e/test_rosacli_machine_pool.go [hunk 1]
```diff
@@ -167,6 +167,26 @@ var _ = Describe("Create machinepool",
 
 			})
 
+		It("validate inputs for create spot machinepool - [id:43252]",
+			labels.Medium,
+			labels.Day1Negative,
```
Comment (radtriste) [inline tests/e2e/test_rosacli_machine_pool.go:172]: machine pool is day2

Comment (jerichokeyne) [reply tests/e2e/test_rosacli_machine_pool.go]: @radtriste I added a `day2-negative` label for it now

Comment (radtriste) [reply tests/e2e/test_rosacli_machine_pool.go]: you did not need to :) `Day2` label was enough. We don't run specific Day2 negative job, they are run as part of Day2

Comment (jerichokeyne) [reply tests/e2e/test_rosacli_machine_pool.go]: Makes sense. Updated the label again. 

Also, do we have some docs somewhere that explain what qualifies as a day1, day2, etc test? I seem to be misjudging what tests qualify for each label

**Comment by yasun1** [comment]
The case is not executed in the Presubmit job. Something is wrong with Prow CI. We say the image is built, but do not find the built log. Compare with the [expected one|https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/pr-logs/pull/openshift_rosa/2068/pull-ci-openshift-rosa-master-e2e-presubmits-pr-rosa-hcp/1792534480230027264/artifacts/] and the [failed one|https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/pr-logs/pull/openshift_rosa/2053/pull-ci-openshift-rosa-master-e2e-presubmits-pr-rosa-hcp/1793198978569867264/artifacts/], we could not see the `build-logs`.

**Comment by yasun1** [comment]
This case is a NonHCP case. We need to run it on the rosa classic cluster.
/test e2e-presubmits-pr-rosa-sts

**Pull request merged**